lua/weapons/gmod_tool/stools/weight_improved.lua

```diff
@@ -67,593 +25,417 @@ local NOTIFY_GENERIC = NOTIFY_GENERIC or 0
 local NOTIFY_ERROR   = NOTIFY_ERROR   or 1
 local NOTIFY_CLEANUP = NOTIFY_CLEANUP or 4
 
-local MIN_WEIGHT      = 0.01      -- the minimum mass that can be set on a physics object (do not change, entities with 0 mass change to 45,678 mass when physgunned)
-local MAX_WEIGHT      = 50000     -- the maximum mass that can be set on a physics object (do not change, this is hardcoded by GMod/Source)
-local MIN_NOTIFY_BITS = 3         -- the minimum number of bits needed to send a NOTIFY enum
-local NOTIFY_DURATION = 5         -- the number of seconds to display notifications
-local MISSING_WEIGHT  = -1234.5   -- the number representing an invalid (not-set) weight, used as default when retrieving networked weights
+local MIN_WEIGHT      = 0.01
+local MAX_WEIGHT      = 50000
+local MIN_NOTIFY_BITS = 3
+local NOTIFY_DURATION = 5
+local MISSING_WEIGHT  = -1234.5
 
 local weightLabels = {
-	original = L(prefix.."hud_original"),
-	modified = L(prefix.."hud_modified")
+    original = "Original",
+    modified = "Modified"
 }
 
---[[--------------------------------------------------------------------------
--- Tool Settings
---------------------------------------------------------------------------]]--
-
 TOOL.Category = "Construction"
-TOOL.Name     = L(prefix.."name")
-
+TOOL.Name     = "Weight - Improved"
 TOOL.Information = {
-	"left",
-	"right",
-	"reload"
+    "left",
+    "right",
+    "reload"
 }
 
-TOOL.ClientConVar[ "mass" ]            = "1"
-TOOL.ClientConVar[ "tooltip_show" ]    = "0"
-TOOL.ClientConVar[ "tooltip_legacy" ]  = "0"
-TOOL.ClientConVar[ "tooltip_scale" ]   = "24"
-TOOL.ClientConVar[ "colorscale" ]      = "1"
-TOOL.ClientConVar[ "rounded" ]         = "1"
-TOOL.ClientConVar[ "decimals" ]        = "2"
-TOOL.ClientConVar[ "notifs"]           = "1"
-TOOL.ClientConVar[ "notifs_sound" ]    = "1"
-
---[[--------------------------------------------------------------------------
--- Convenience Functions
---------------------------------------------------------------------------]]--
-
-function TOOL:GetMass()                return self:GetClientNumber( "mass" )                end
+TOOL.ClientConVar["mass"] = "1"
+TOOL.ClientConVar["tooltip_show"] = "0"
+TOOL.ClientConVar["tooltip_legacy"] = "0"
+TOOL.ClientConVar["tooltip_scale"] = "24"
+TOOL.ClientConVar["colorscale"] = "1"
+TOOL.ClientConVar["rounded"] = "1"
+TOOL.ClientConVar["decimals"] = "2"
+TOOL.ClientConVar["notifs"] = "1"
+TOOL.ClientConVar["notifs_sound"] = "1"
```

Comment:
If we're being opinionated...
```suggestion
TOOL.ClientConVar.mass = "1"
TOOL.ClientConVar.tooltip_show = "0"
TOOL.ClientConVar.tooltip_legacy = "0"
TOOL.ClientConVar.tooltip_scale = "24"
TOOL.ClientConVar.colorscale = "1"
TOOL.ClientConVar.rounded = "1"
TOOL.ClientConVar.decimals = "2"
TOOL.ClientConVar.notifs = "1"
TOOL.ClientConVar.notifs_sound = "1"
```